backend/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "nodemon dist/index.js",
```

can we add compilation in watch mode before running nodemon something like - 
```suggestion
    "dev": "tsc -w && nodemon dist/index.js",
```

And please add a Readme.